airbyte-integrations/connectors/source-domo/.vscode/launch.json

```diff
@@ -0,0 +1,49 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Check",
+            "type": "python",
+            "request": "launch",
+            "program": "main.py",
+            "console": "integratedTerminal",
+            "args": [
+                "check",
+                "--config",
+                "./secrets/config.json"
+            ],
+            "justMyCode": false
+        },
+        {
+            "name": "Discover",
+            "type": "python",
+            "request": "launch",
+            "program": "main.py",
+            "console": "integratedTerminal",
+            "args": [
+                "discover",
+                "--config",
+                "./secrets/config.json"
+            ],
+            "justMyCode": false
+        },
+        {
+            "name": "Read",
+            "type": "python",
+            "request": "launch",
+            "program": "main.py",
+            "console": "integratedTerminal",
+            "args": [
+                "read",
+                "--config",
+                "./secrets/config.json",
+                "--catalog",
+                "./catalogs/configured_catalog.json"
+            ],
+            "justMyCode": false
+        }
+    ]
+}
```

Comment:
I think this file accidentally made it in to the repo. I saw the `.vscode` got added to `.gitignore` but this one made it in anyway.